.github/workflows/release.yml

```diff
@@ -39,11 +39,9 @@ jobs:
               --output-indicator-new=! CHANGELOG.md | egrep '^!' | awk -F'^[!]' '{print $2}' | sed -e 's/\n/$0A/g'
           )
 
-          echo "chore(release): v$Version" > /tmp/pr.txt
-          echo "" >> /tmp/pr.txt
           echo "$ReleaseNotes" >> /tmp/pr.txt
           echo "" >> /tmp/pr.txt
           echo "This PR was opened by a robot :robot: :tada:" >> /tmp/pr.txt
-          hub pull-request --file /tmp/pr.txt --base master
+          gh pr create --title "chore(release): v$Version" --body-file "/tmp/pr.txt" --base release
```

Comment:
Is it intentional that this changes the base from `master` to `release`? Do the two commands use `--base` to mean different things?